StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/CommentDto.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class CommentDto
+    {
+        public string? Content { get; set; }
+
+        public DateTime CommentDate { get; set; }
```

Comment:
It difficult to understand what does it mean
Date when comment was created? => CreatedAt
Published? => PublishedAt
and so on...